release/ray_release/cluster_manager/cluster_manager.py

```diff
@@ -51,6 +51,9 @@ def set_cluster_env(self, cluster_env: Dict[str, Any]):
         self.cluster_env["env_vars"]["RAY_bootstrap_with_gcs"] = "1"
         self.cluster_env["env_vars"]["RAY_USAGE_STATS_ENABLED"] = "1"
         self.cluster_env["env_vars"]["RAY_USAGE_STATS_SOURCE"] = "nightly-tests"
+        self.cluster_env["env_vars"][
+            "RAY_USAGE_STATS_EXTRA_TAGS"
+        ] = f"test_name={self.test_name};smoke_test={self.smoke_test}"
```

Comment:
This (and the ones above) still overwrite what is set in the cluster env, correct? can we add a comment here to separate these two groups?